esmvaltool/diag_scripts/climate_metrics/zec.py

```diff
@@ -233,7 +233,8 @@ def main(cfg):
     plot_zec_timeseries(zec, cfg)
 
     # Calculate ZEC_X
```

Comment:
Thanks for updating `zec_year` variable name - clearer for me now. It would be helpful to update this comment also to explain the link between `zec_year` that the user sees, and the `zec_x` variable in the loop below.